### PR TITLE
Add explicit test/CI support for python 3

### PR DESCRIPTION
**Pull request by CloudNiner**
## Overview

The django-amazon-ses library does not officially support python 3. This PR adds current python 3 versions to the build matrix so that we can begin supporting it. It also notes python 3 support in setup.py. 

No code changes were necessary to get django-amazon-ses to work on python 3.

### CODE REVIEW

**Comment by CloudNiner** [comment]
My first effort with `tox`. Happy to clean up the changes there if there's a better way to go about this.

File: tox.ini [hunk 1]
```diff
@@ -1,22 +1,27 @@
 [tox]
-envlist = py27
+envlist = py27, py34, py35, py36
+recreate = True
```
Comment (hectcastro) [inline tox.ini]: Do you know why this option is necessary? It seems as though this force recreates the virtual environment.

**Comment by hectcastro** [comment]
Thanks for taking the time to add Python 3.x support to this module.

In reviewing this PR, I started looking at the test suites of other popular Django modules. It looks like the Django REST Framework uses a Travis build matrix to test Python and Django versions, so I opened an issue (https://github.com/azavea/django-amazon-ses/issues/5) to do something like that here. I think making that happen, and adding Python 3 support, are different enough that it can be tackled separately.

Any hesitation around doing that?

File: tox.ini (hunk 1 above)
Comment (CloudNiner) [reply tox.ini]: Whoops. No I don't think it was necessary. I'll remove and test.

**Comment by CloudNiner** [comment]
Nope no hesitation, I think the strategy outlined in #5 makes sense. DRF seems to have a sensible, straightforward setup.

**Review by hectcastro — APPROVED** [review]
👍

**Pull request merged**